types/leaflet/index.d.ts

```diff
@@ -728,10 +734,13 @@ export class LayerGroup<P = any> extends Layer {
     feature?: geojson.FeatureCollection<geojson.GeometryObject, P> | geojson.Feature<geojson.MultiPoint, P> | geojson.GeometryCollection;
 }
 
+// @factory L.layerGroup(layers?: Layer[], options?: Object)
 /**
- * Create a layer group, optionally given an initial set of layers.
+ * Create a layer group, optionally given an initial set of layers and an `options` object.
  */
-export function layerGroup(layers: Layer[]): LayerGroup;
+export function layerGroup(layers?: Layer[]): LayerGroup;
```

Comment:
This line is optional as long as you also add line 743!

When options are optional it is absolute clear that you can also pass it without it...